.github/workflows/polyglot-build.yml

```
@@ -11,13 +11,15 @@ on:
     branches: [main]
-
-permissions: {}
+permissions:
+  contents: read
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
+permissions: {}
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:14,col:1

(syntax-check)